src/content/resources/link-token-contracts.mdx

```diff
@@ -306,9 +306,9 @@ Testnet LINK is available at [faucets.chain.link](https://faucets.chain.link/sep
 
 ### Optimism Goerli testnet
 
-Testnet ETH is used to pay for transactions on Optimism. Use the [Optimism Bridge](https://app.optimism.io/bridge) to transfer testnet ETH and LINK from Ethereum Goerli to Optimistim Goerli.
+Testnet ETH is used to pay for transactions on Optimism. Use the [Optimism Bridge](https://app.optimism.io/bridge) to transfer testnet ETH from Ethereum Goerli to Optimistim Goerli.
```

Comment:
The Optimism Bridge doesn't support Goerli anymore. As does [OP official faucet](https://app.optimism.io/faucet). Should we consider OP Goerli as deprecated in our doc? It looks like OP Goerli's maintenance will be decommissioned soon.